build/build.py

```diff
@@ -1712,6 +1714,14 @@ def printHelp():
     print("  --name=Validator.nu        -- Sets service name")
     print("  --bind-address=0.0.0.0     -- Sets server bind address")
     print("  --port=8888                -- Sets server port number")
+    print("  --allowed-address-type=<value>")
+    print("                                Set the value to specify")
+    print("                                what URLs are accepted.")
+    print("                                Possible values:")
+    print("                                - 'all': Accept all URLs (default)")
+    print("                                - 'same-origin': Accept only")
+    print("                                  same-origin URLs")
+    print("                                - 'any': Disallow URL input)")
```

Comment:
Actually, shouldn’t this value be `'none'`?

Reply:
@sideshowbarker Thank you for reviewing and telling me correct English.
Yes, I changed just now!

[cc50de6](https://github.com/validator/validator/pull/1638/commits/cc50de6f22cb90746bd892e62815bb4593f6dad8)

Reply:
@sideshowbarker Sorry, It appears that unintended differences remain.
Please wait for fixing...

Reply:
@sideshowbarker 
Thank you for your waitting!
All corrections are done. Please check it!

[5e4d10f](https://github.com/validator/validator/pull/1638/commits/5e4d10f710f8e73e62b4f6b1f8f2fb75667b998c)